keyboards/handwired/franky36/keyboard.json

```diff
@@ -23,8 +23,8 @@
     "url": "https://github.com/avdyushin/franky36",
     "usb": {
         "device_version": "0.2.0",
-        "pid": "0x0001",
-        "vid": "0xFEED"
+        "pid": "0x1337",
```

Comment:
https://pid.codes/pids/

Reply:
Updated to non reserved and requested one: https://github.com/pidcodes/pidcodes.github.com/pull/1047